### ISSUE TITLE
mount a usb stick to a VM

### ISSUE THREAD

**Issue by iliassch** [retspen/webvirtcloud#313]
Hi,

Is there a way to mount to an existing VM , a usb stick that has been physically connected to the server that hosts the VM?

**Comment by catborise** [comment]
@iliassch yes you can mount it but you should use virt-manager. (install virt-manager to hypervisor host)

we do not implement pyhsical dev management of libvirt.

**Comment by iliassch** [comment]
Hi @catborise , thanks for your answer. Yes i have tried it with a locally installed program(either virt-manager or virt-viewer) through SSH and X11 and it worked. If you change display to SPICE server instead of VNC , there is an active usb redirect button to both programs. I just wondering how you can do it remotely through webvirtcloud  or without ssh access ,  connect to the SPICE server , retrieve physically connected usb devices and mount them to the VM.

Possibly with an extension of noVNC  or something , to send a command to retrieve devices and then select the proper one.
